docker-compose.dev.yml

```diff
@@ -0,0 +1,28 @@
+load_balancer:
+    image: tutum/haproxy
+    links:
+        - web
+    ports:
+        - "8000:80"
+
+web:
+    image: savaslabs/squabble
+    environment:
+      - APP_ENV=local
+      - APP_DEBUG=true
+      - APP_KEY=abracadabra
+      - CACHE_DRIVER=file
+      - SESSION_DRIVER=file
+      - APP_LOCALE=en
+      - APP_FALLBACK_LOCALE=en
+      - DB_CONNECTION=sqlite
+      - DB_HOST=localhost
+      - NOCAPTCHA=owl
+      - MAIL_DRIVER=log
+      - BASEURL=localhost:8000
```

Comment:
In anticipation of Dockerizing our Savas Labs website app, we could change this to `BASEURL=local.savaslabs.com` or something like that